tests/testfiles/lunamark-markdown/gfm-auto-identifiers.test

```diff
@@ -0,0 +1,110 @@
+\def\markdownOptionGfmAutoIdentifiers{true}
```

Comment:
Much like in `auto-identifiers-with-attributes.test`, we should test the interaction between explicitly written identifiers and automatic identifiers by enabling the `headerAttributes` option here and testing it below.